src/components/DescriptionPopup.js

```diff
@@ -1,129 +1,173 @@
       });
-    });
-    }
+  };
 
-    rateMovie = (value) => {
-      const { movie } = this.props
-      axios.post(
-        this.state.ratings_api+"/rate/",
-        {movie_name: movie.Title, engine: movie.Source, ip_address: this.props.ip_address, score: value}
-      )
+  rateMovie = value => {
+    const { movie } = this.props;
+    axios
+      .post(this.state.ratings_api + "/rate/", {
+        movie_name: movie.Title,
+        engine: movie.Source,
+        ip_address: this.props.ip_address,
+        score: value
+      })
       .then(res => {
         if (res.data !== null) {
           this.setState({
             ip_rating: res.data.score
-          })
+          });
         }
         // retrieve average to force rerender
         this.getAverage();
-    })
-    .catch(err => {
-      this.setState({
-        error: true
+      })
+      .catch(err => {
+        this.setState({
+          error: true
+        });
       });
-    });
-    }
+  };
+
+  render() {
+    return (
+      <Modal
+        show={this.props.show}
+        onHide={() => this.props.onHide()}
+        size="lg"
+        aria-labelledby="contained-modal-title-vcenter"
+        dialogClassName="dialog-theme"
+        data-tour="my-fifth-step"
+        centered
+      >
+        <Modal.Body className="gophie-modal">
+          <section className="gophie-modal__img">
+            <img
+              src={
+                this.props.movie.CoverPhotoLink === ""
+                  ? "No image"
+                  : this.props.movie.CoverPhotoLink
+              }
+              alt={this.props.movie.Title}
+            />
+          </section>
+
+          <section className="gophie-modal__body">
+            <Modal.Header className="gophie-modal__body--header" closeButton>
+              <Modal.Title id="contained-modal-title-vcenter">
+                {this.props.movie.Title}
+              </Modal.Title>
+            </Modal.Header>
+            <section className="gophie-modal__body--body">
+              <div className="gophie-modal-rating-container">
+                <div className="gophie-modal-rating-container__average">
+                  <Rating
+                    value={Math.round(
+                      this.state.ratings.average_ratings
+                        ? this.state.ratings.average_ratings
+                        : 0
+                    )}
+                    max={5}
+                    readOnly={true}
+                  />
+
+                  <div className="gophie-modal-rating-container__average--container">
+                    <div className="gophie-modal-rating-container__average--container-item-1">
+                      <p>
+                        {this.state.ratings.average_ratings
+                          ? Math.round(
+                              this.state.ratings.average_ratings * 10
+                            ) / 10
+                          : 0}
+                      </p>
+                      <p>/5</p>
```

Comment:
I think people need to also know how many people have rated the movie. That's an important metric when deciding if the average you see is a good reflection